integration/admin/admin_integration_test.go

```diff
@@ -52,10 +52,7 @@ func TestAdminServer_CreateTree(t *testing.T) {
 	invalidTree := proto.Clone(testonly.LogTree).(*trillian.Tree)
 	invalidTree.TreeState = trillian.TreeState_UNKNOWN_TREE_STATE
 
-	timestamp, err := ptypes.TimestampProto(time.Unix(1000, 0))
-	if err != nil {
-		t.Fatalf("TimestampProto() returned err = %v", err)
-	}
+	timestamp := timestamppb.New(time.Unix(1000, 0))
```

Comment:
If we want to preserve the error handling then it should call CheckValid on timestamp. However as this is a test it might not be worth it.